examples/kanban/src/main/java/com/example/kanbanapp/KanbanBoardViewModel.kt

```diff
@@ -24,7 +24,9 @@ class KanbanBoardViewModel(private val client: Client) : ViewModel() {
 
     init {
         viewModelScope.launch {
-            if (client.activateAsync().await() && client.attachAsync(document).await()) {
+            if (client.activateAsync().await().isSuccess &&
+                client.attachAsync(document).await().isFailure
```

Comment:
I think it should be `client.attachAsync(document).await().isSuccess`

Reply:
done [bb70e2e](https://github.com/yorkie-team/yorkie-android-sdk/pull/179/commits/bb70e2ed131470b26e9f69fce4560c936a7035ec)